.github/workflows/master.yml

```diff
@@ -1,6 +1,9 @@
 name: master
 
 on:
+  push:
+    branches:
+    - master
```

Comment:
For Github Actions running for push event, where I can look at the action logs? Still in "Check" tab of PRs? When will it be triggered? Merging?

Reply:
We can see at two places.
1. At commit log. At every commit log, the green check mark is attached.
- https://github.com/apache/spark/commits/master
2. At commit info,
- https://github.com/apache/spark/commit/10a65ee9b42fd544e4698267e0cd16711ed92104

Currently, it's disabled yesterday. So, recent commits do not have that.